extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -1,20 +1,22 @@
 <?php
 /**
  * Class definition for \Wikia\Search\MediaWikiService
+ *
  * @author relwell
  */
 namespace Wikia\Search;
+
 use Wikia\Search\Result\StaleResultException;
 
 /**
  * Encapsulates MediaWiki functionalities.
  * This will allow us to abstract our behavior away from MediaWiki if we want.
  * Public functions should not return instances of classes defined in MediaWiki core.
+ *
  * @author relwell
  * @package Search
  */
-class MediaWikiService
-{
+class MediaWikiService {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "MediaWikiService" has 77 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)